### PR TITLE
LPS-149017 [CP 2.0] Verify if analytics cloud form was already submitted on submit form action and InProgress activation status

### PR DESCRIPTION
**Pull request by igordmf**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/AnalyticsCloudModal/index.js [hunk 1]
```diff
@@ -39,16 +41,33 @@ const AnalyticsCloudModal = ({
...
 
+	const submittedModalTexts = {
+		paragraph:
+			'Return to the product activation page to view the current Activation Status',
+		subtitle: `We'll need a few details to finish building your Analytics Cloud workspace(s).`,
+		text:
+			'Another user already submitted the Analytics Cloud activation request.',
+		title: 'Set up Analytics Cloud',
+	};
+
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/AnalyticsCloudModal/index.js:60]: Hi @igordmf, constant values should be good created before of component as not depend the component render

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js [hunk 2]
```diff
@@ -43,10 +44,23 @@ const SetupDXPCloudModal = ({
...
+	const submittedModalTexts = {
+		paragraph:
+			'Return to the product activation page to view the current Activation Status',
+		subtitle: `We'll need a few details to finish building your DXP
+		environment(s).`,
+		text:
+			'Another user already submitted the DXP Cloud activation request.',
+		title: 'Set up DXP Cloud',
+	};
+
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js:56]: Hi @igordmf, constant values should be good created before of component as not depend the component render

**Review by thaynam — APPROVED** [review]

**Review by claricebrasil — APPROVED** [review]
Great job, Igor! I just miss your PR description, just telling what this PR is about.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.js [hunk 3]
```diff
@@ -105,51 +107,78 @@ const SetupAnalyticsCloudPage = ({
...
+					variables: {
+						accountSubscriptionGroup: {
+							activationStatus: STATUS_TAG_TYPE_NAMES.inProgress,
+						},
+						id: subscriptionGroupId,
+					},
+				});
+
+				await Promise.all(
+					analyticsCloud?.incidentReportContact.map(({email}) => {
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.js:166]:
```suggestion
					analyticsCloud?.incidentReportContact?.map(({email}) => {
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.js [hunk 4]
```diff
@@ -105,51 +107,78 @@ const SetupAnalyticsCloudPage = ({
...
-					id: subscriptionGroupId,
+					scopeKey: Liferay.ThemeDisplay.getScopeGroupId(),
 				},
 			});
-		}
 
-		handlePage();
+			if (data) {
+				const analyticsCloudWorkspaceId =
+					data?.c?.createAnalyticsCloudWorkspace
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.js:152]:
```suggestion
					data.c?.createAnalyticsCloudWorkspace
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]
Hi @igordmf. Good job man, I made some suggestions basic about constant declaration position. :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/AnalyticsCloudModal/index.js (hunk 1 above)
Comment (igordmf) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/AnalyticsCloudModal/index.js]: Done! Thank you!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js (hunk 2 above)
Comment (igordmf) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js]: Done! Thank you!

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**